.env.example

```diff
@@ -9,6 +9,7 @@ REACT_APP_LOGROCKET_APP_ID=
 REACT_APP_MIXPANEL_PROJECT_TOKEN=
 REACT_APP_WS_RELAY_URL=
 REACT_APP_CODE_CHECK_URL=
+REACT_APP_FACEBOOK_APP_ID=
```

Comment:
NOTE: We should set this in our env

Reply:
@oleksii-front I have deleted your comment here, please don't share this value on our public code repo.